File: backend/GymRecommender/models/rating.py
from django.db import models
from .account import Account
from .gym import Gym

class Rating(models.Model):
    id = models.UUIDField(primary_key=True)
    created_at = models.DateTimeField()
    changed_at = models.DateTimeField()
    rating = models.IntegerField()
    user = models.ForeignKey(Account, models.DO_NOTHING)
    gym = models.ForeignKey(Gym, models.DO_NOTHING)

    class Meta:
        managed = False
        db_table = 'rating'
        unique_together = (('user', 'gym'),)

File: backend/Models/Rating.cs
using System;
using System.Collections.Generic;

namespace backend.Models;

public partial class Rating
{
    public Guid Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime ChangedAt { get; set; }

    public int Rating1 { get; set; }

    public Guid UserId { get; set; }

    public Guid GymId { get; set; }

    public virtual Gym Gym { get; set; } = null!;

    public virtual Account User { get; set; } = null!;
}
